srcpkgs/datovka/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'datovka'
+pkgname=datovka
+version=4.21.1
+revision=1
+archs="~*-musl"
```

Comment:
Please move `archs` to `libdatovka`.

My apologies for missing it earlier, did not pay attention to the package name.